orchestra/actions/configure.py

```diff
@@ -20,7 +20,8 @@ def _run(self, cmdline_args):
             os.remove(self._configure_successful_path)
         elif os.path.exists(self.environment["BUILD_DIR"]):
             logger.warning("Previous configure probably failed, running configure script in a dirty environment")
-            logger.warning(f"You might want to delete the build directory (use `orchestra clean`)")
+            logger.warning(f"You might want to delete the build directory "
+                           f"(use `orchestra clean {self.build.qualified_name}`)")
```

Comment:
Maybe reports something copy pastable with a triple click (i.e., select the whole line).

Reply:
Unfortunately `loguru` already decorates the lines so copying the whole line won't work.
I quoted the command with backticks to help to visually identify the command and hopefully the terminal emulator to select the right portion.